visual_odometry/LaserDot[] laser_dots
